src/index.html

```diff
@@ -12,7 +12,39 @@
       href="styles/index.scss"
     />
```

Comment:
Link to an SCSS file won't work directly in HTML. You should compile your SCSS to CSS and then link the compiled CSS file.